modules/social_features/social_branding/tests/src/Unit/ColorTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\Tests\social_branding\Unit;
+
+use Drupal\social_branding\Wrappers\Color;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Color test.
+ *
+ * @coversDefaultClass \Drupal\social_branding\Wrappers\Color
+ * @group social_branding
+ */
+class ColorTest extends UnitTestCase {
```

Comment:
Nice unit tests! Some tweaks though:

- Please use `self::assertEquals` since it's a static method (the following PHPStorm plugin can help with this: https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended- )
- I believe (but could be wrong) that `assertEquals` already does type-strict checking
- You're using return types in the color class, so your class is already strongly typed 🎉 so the type tests aren't needed :D